rafthttp/http.go

```diff
@@ -72,6 +74,15 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := checkVersionCompability(serverVersion(r.Header), minClusterVersion(r.Header)); err != nil {
+		log.Printf("rafthttp: request ignored due to %v", err)
+		http.Error(w, errMsgIncompatibleVersion, http.StatusForbidden)
+		return
+	}
+	if err := checkClusterVersionCompatiblity(r.Header.Get("X-Cluster-Version")); err != nil {
+		log.Printf("rafthttp: received request that %v", err)
```

Comment:
can you give me an example that this check fails when the check at line 77 passes?